### PR TITLE
Add a hotkey to spawn reminder tokens.

### LINKED ISSUE #131: Suggestion: add hotkeys for spirit markers & reminders.
I've had a few requests for this when playing in public lobbies.

### PR DESCRIPTION
**Pull request by pyrrhicPachyderm**
Resolve #131.

### CODE REVIEW

**Comment by pyrrhicPachyderm** [comment]
I keep getting bugged about adding this. I was going to wait until #130 was resolved, but I realise that's a big change and so it might be a while yet, so here's this first. I'll update #130 to be compatible with this if this gets merged.

File: script.lua [hunk 1]
```diff
@@ -7729,122 +7732,82 @@ function onObjectDestroy(obj)
...
+        maskImage = "CardMask"
+    else
+        return
+    end
+
+    if imageURL == nil then
+        return
+    end
+
+    if isMarker then
```
Comment (iakona) [inline script.lua:7781]: could we just drop the `isMarker` parameter and move this into the above conditional blocks?

File: script.lua [hunk 2]
```diff
@@ -7729,122 +7732,82 @@ function onObjectDestroy(obj)
...
+        imageURL = objData.CustomImage.ImageSecondaryURL
+        if obj.hasTag("Lower Spirit Image") then
+            panelWidth, panelHeight, panelX, panelY = 378, 252, -97, -33
+            maskImage = "SpiritMask2"
+        else
+            panelWidth, panelHeight, panelX, panelY = 480, 320, -127, 46
+            maskImage = "SpiritMask"
+        end
+    elseif obj.type == "Card" then
+        position = position + Vector(0, 0.2, 0)
```
Comment (iakona) [inline script.lua:7769]: not sure if this is intentional, but the value before was `0.02`

Comment (pyrrhicPachyderm) [reply script.lua]: Whoops, good catch! Fixed.

File: script.lua (hunk 1 above)
Comment (pyrrhicPachyderm) [reply script.lua:7781]: Whether or not we want a marker isn't a property of the object we're spawning it for: if we're acting on a spirit panel, we might be asking for a reminder or for a spirit marker. So this definitely has to be a parameter.

I considered having the function abort if it was called on a card with `isMarker = true`, but the code would still work without errors in this case, so there's no real reason to forbid it (it just never gets called that way currently).

Comment (iakona) [reply script.lua:7781]: ah sorry the diff for this PR is not very pretty, need to have like 4 windows up to see what is really going on here. i'll have to take another look at this when it's not super late xD

Comment (pyrrhicPachyderm) [reply script.lua:7781]: Yeah, sorry about the diff; I noticed that myself when I was adding it. Really what I did was write one new function and turn a large chunk inside three other functions into just calls to that function. I'll see if I can find a way to make it display the diff like that for easier reading.

Comment (pyrrhicPachyderm) [reply script.lua:7781]: If you pull this branch locally and run `git diff --anchored="function applyPowerCardContextMenuItems(card)" beta reminder-hotkey`, then it'll display the diff in a *useful* format. I don't know if there's any way to make GitHub do the same.

**Pull request merged**